libcircle.spec.0.3

```diff
@@ -0,0 +1,171 @@
+Name:    libcircle
```

Comment:
Why is this file in the repo?

Reply:
Just in case it was needed for a v0.3 release.